pkg/sparrow/controller.go

```diff
@@ -125,7 +119,7 @@ func (cc *ChecksController) Reconcile(ctx context.Context, cfg runtime.Config) {
 
 	// Unregister checks not in the new config
 	for _, c := range unregList {
-		err = cc.UnregisterCheck(ctx, c)
+		cc.UnregisterCheck(ctx, c)
 		if err != nil {
 			log.ErrorContext(ctx, "Failed to unregister check", "check", c.Name(), "error", err)
 		}
```

Comment:
This can also be removed

Reply:
Removed it, thanks for the review. The `RegistrerCheck` also never seems to return an error, it just pumps everything down the error channel... I'll change that in a later PR.